src/env.js

```diff
@@ -1,9 +1,21 @@
 const DEFAULT_INSTANCE_ID = 1;
 
-if (process.env.NODE_ENV === 'production') {
-  process.env.CONSOLE_LOGGING = 'false';
+if (process.env.NODE_ENV !== 'production') {
+  process.env.CONSOLE_LOGGING = 'true';
 }
 
+// process.env.BUGSNAG_LOGGING = 'true';
+// process.env.LOGENTRIES_LOGGING = 'true';
+// process.env.MIN_LOG_LEVEL = 'warn';
+// process.env.MIN_LOG_LEVEL_CONSOLE = 'info';
+// process.env.MIN_LOG_LEVEL_LOGENTRIES = 'error';
+// process.env.MIN_LOG_LEVEL_BUGSNAG = 'error';
+// process.env.DEFAULT_LOG_LEVEL = 'info';
+// process.env.LOG_TIMESTAMP = 'true';
+// process.env.LOG_ENVIRONMENT = 'true';
+// process.env.LOG_LEVEL = 'true';
+// process.env.LOG_REQID = 'true';
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.dialonce.io/coding_rules#rule_key=javascript%3ACommentedCode)